lib/tasks/dynflow.rake

```diff
@@ -10,4 +10,11 @@ END_DESC
   task :executor => :environment do
     Dynflow::Rails::Daemon.new.run
   end
+
+  task :migrate => :environment do
+    world = OpenStruct.new(:config => OpenStruct.new(:queues => {}))
+    config = Dynflow::Rails::Configuration.new
+    # Persistence initialization automatically migrates the db
+    config.send(:initialize_persistence, world)
```

Comment:
If this is a prerequisite to the migration script, shouldn't it just be added to it?

Reply:
It is a prerequisite the same way as regular `db:migrate` is. Given the rake task expects both DBs to have the same schema, the current approach makes sense to me.

Reply:
What exactly this does? I am not following why this needs to be executed prior `prod2dev`.

Reply:
It performs the migration of Dynflow tables, which are not handled by regular `db:migrate`, because Dynflow doesn't use active record

Reply:
would it be a noop in case it ran already? if so, i'd add that to the migration script just to make sure users don't miss it and we don't ask them for an extra manual step

Reply:
Yeah needs to be idempotent.

Reply:
It is. Changed prod2dev to run this task